### DISCUSSION TITLE
Adding to shared album

### DISCUSSION THREAD

**Discussion by micogg** [immich-app/immich#920] [General]
Sorry if I missed this discussion, but is it possible for two people to auto-backup to the same shared album?

I have a folder backing up with family pictures.  Can my wife set her phone to back to the same album or do people have to access two separate albums?

**Comment by micogg** [comment]
Following up on this.  I gave access for extended family to a specific shared album.  I want to backup photos from my wife's phone, but I can't find a way to add them to the existing shared album.  I'd rather not have to give them access to two shared albums to access the same categorized images.  Is it currently possible?

**Reply by alextran1502** [reply]
Hello, I just want to clarify that you cannot add an existing uploaded photo to a shared album that you have access to, is this correct?

**Reply by micogg** [reply]
I am curious if you can auto-upload directly to a shared album that you do not own.

I created a shared album that I upload family pictures to from my phone. 
I would like to auto-upload pictures from my wife's phone to the same shared album so we don't have to share two different albums with people.

Is it currently possible for two people to auto-upload to a single shared album?

**Reply by alextran1502** [reply]
That feature would be "shared with partners", we are aiming to provide this feature in the future

**Comment by AngelPone** [comment]
Vote for this.
Now we can upload photos to the own library using the cli and adding them to a shared album manually.
It's would be great to upload to existing shared album directly. 
The CLI `immich upload` has a `--album` option to automatically upload to a existing own album or create and upload to a non-existing album. 
But if a local folder named same as a shared album and the album does not exist in your own library, it won't upload to the shared album. Instead, the cli would create a new album in your own library.

I wonder If we can add a option such as `--prefer-shared-album`. If the local folder exists in the shared albums and own library, upload to shared album.
